src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/uninstrumented-entities-service-maps.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Uninstrumented entities in service maps'
+tags:
+  - Understand dependencies
+  - Understand system dependencies
+  - Uninstrumented entities
+  - Service maps
+metaDescription: Uninstrumented entities are created by New Relic and can be easily instrumented by the customer
+---
+import uninstrumentedEntityServiceMap from 'images/accounts_diagram_NR-parent-child-account-structure-.png'
```

Comment:
s/uninstrumentedEntityServiceMap/`uninstrumentedEntityServiceMap `.
Also is the `from` path correct?

Reply:
no. It has been corrected